express-get-json/server.js

```diff
@@ -0,0 +1,46 @@
+import express from 'express';
+
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const getGrades = (req, res, next) => {
+  const json = [];
+  for (const key in grades) {
+    json.push(grades[key]);
+  }
+  next(sendBack(res, json));
+};
+
+const sendBack = (res, info) => {
+  res.json(info);
+};
```

Comment:
This is reusable, which is nice, but it's very easy to just call `res.json(info)` everywhere. The fact you are returning json is pretty baked into the protocol.